libs/components/lists/src/lib/modules/repeater/repeater.component.ts

```diff
@@ -53,6 +53,7 @@ export class SkyRepeaterComponent
    * Specifies an ARIA label for the repeater list.
    * This sets the repeater list's `aria-label` attribute
    * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * For more information about the `aria-label` attribute, see the [WAI-ARIA Definitions of States and Properties](https://www.w3.org/TR/wai-aria/#aria-label).
    * @default "List of items"
    */
   @Input()
```

Comment:
Do we need to add anything to the SkyRepeaterItemComponent's `itemName` property since it also is also related to ARIA. Seems like it might need the "For more information" sentence and a link to https://www.w3.org/TR/wai-aria/#aria-label.

Reply:
Updated